src/views/pages/items.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import ItemStore from '../../store/item-store.js';
+import { viewModel } from '../../lib/api-client.js'
+import { getModelsForNewItem } from '../../lib/api-client';
+import Item from '../components/item.jsx';
+export default class Models extends ListenerComponent {
+
+    constructor() {
```

Comment:
Overriding the constructor isn't necessary if you aren't going to do anything with it.